prepare-vms/settings/swarm.yaml

```diff
@@ -1,7 +1,10 @@
 # This file is passed by trainer-cli to scripts/ips-txt-to-html.py
 
+# Workshop name for cards, either fundamentals, kube101, or swarm
+type: swarm
```

Comment:
Do we need that one?

Reply:
(Perhaps the associated change in cards.html is in another PR, I'll keep looking :-))